cicerone/spec/callback.py

```diff
@@ -0,0 +1,62 @@
+"""Callback model for OpenAPI callbacks.
+
+References:
+- OpenAPI 3.x Callback Object: https://spec.openapis.org/oas/v3.1.0#callback-object
+
+A Callback Object is a map of possible out-of-band requests and responses
+that may be initiated by the API provider and expected to be handled by the API consumer.
+The key is a runtime expression that identifies a URL to use for the callback operation.
+The value is a Path Item Object or a reference to one.
+"""
+
+from __future__ import annotations
+
+from typing import Any
+
+from pydantic import BaseModel, Field
+
+from cicerone.spec.path_item import PathItem
+
+
+class Callback(BaseModel):
+    """Represents an OpenAPI Callback Object.
+
+    A callback is a map of runtime expressions to Path Item Objects.
+    Each runtime expression defines a URL where a callback request will be sent.
+    """
+
+    # Allow extra fields to support vendor extensions
+    model_config = {"extra": "allow"}
+
+    # Callbacks are a dict of expression -> PathItem
+    expressions: dict[str, PathItem] = Field(default_factory=dict)
+
+    @classmethod
+    def from_dict(cls, data: dict[str, Any]) -> Callback:
+        """Create a Callback from a dictionary.
+
+        Args:
+            data: Dictionary containing callback expressions mapping to Path Items
+
+        Returns:
+            Callback object with expressions parsed as PathItem objects
+        """
+        from cicerone.spec.path_item import PathItem
```

Comment:
Re-imported here? Why not use the top-level import? Justify the reason for this being here, and if it isn't correct remove it.